application/features/authentication/authentication.feature

```diff
@@ -0,0 +1,30 @@
+Feature: Authentication
+  In order to identify a specific user
+  As an authorized user
+  I need to be able to authenticate a user based on certain credentials
+
+  Scenario: Create an authentication for a known user with a matching password
+    Given I am authorized
+      And I receive a username
+      And I receive a password
+    When I verify the password matches the associated password
+    Then I will respond positively
+
+  Scenario: Attempt to create an authentication for an unknown user
+    When I realize the username does not exist in my system
```

Comment:
most definitely, sorry I didn't make it clear that I hadn't gotten to these at all yet...it was simply a scratchpad for thoughts at the time.  These will be built out in https://github.com/silinternational/idp-id-broker/issues/7